src/Npgsql/Internal/NpgsqlConnector.cs

```diff
@@ -2497,7 +2503,7 @@ internal async Task ExecuteInternalCommand(byte[] data, bool async, Cancellation
     {
         Debug.Assert(State != ConnectorState.Ready, "Forgot to start a user action...");
 
-        Log.Trace("Executing internal pregenerated command", Id);
+        Logger.LogTrace("Executing internal pregenerated command", Id);
```

Comment:
Some strings in this PR have arguments but not corresponding parameter in the template.

Reply:
That's intentional - the idea is to report certain structured fields but without necessarily including them in the message (see https://github.com/dotnet/runtime/issues/35995), since it's not always appropriate to pack all possible data into that string. The connection ID is a good example - I'm not sure we should overload textual output for every single message with the ID (because in many/most cases that doesn't matter), but the data should be there in case someone wants to filter by a specific connection id, for example.

Any thoughts on this? Do you think it's problematic practice

Reply:
All good, wanted to check if it was intentional.